### ISSUE TITLE
Reported: Recurring renewals not being included in the iOS app reports

### ISSUE THREAD

**Issue by cklosowski** [easydigitaldownloads/edd-ios#112]
Reported here: https://twitter.com/jackmcconnell/status/1041933643982295040?s=12

I haven't confirmed yet (no iOS device anymore), but if you could take a look that'd be awesome. :heart:

**Comment by jackmcconnell** [comment]
@cklosowski Happy to provide more info on this if needed:
It was working fine before a recent update to the Recurring Payments add-on but since then (possibly since early September), it's stopped adding recurring payments to the current monthly total on the graph screen of the iOS app and they're also not showing in the sales tab. All other types of sales (e.g. new sales) seem to be up-to-date and showing as expected in the Sales tab but they too don't seem to be adding to the Current Total on the graph screen. Could also be an issue with a recent API update I guess, if there was one.

**Comment by cklosowski** [comment]
@jackmcconnell At this point I've looked through both the Recurring releases as well as the EDD Core releases and there haven't been any changes in the last 2-3 months that should have affected the calculation of overall earnings.

I had Pippin test this on his iOS app and he is seeing all the proper numbers. Have you tried disabling other plugins to see if something is incorrectly modifying our order statuses for those queries.

**Comment by jackmcconnell** [comment]
@cklosowski Hmm. Odd. I just un-installed the app, installed it again, and re-inputted my API credentials. This updated the current total on the Dashboard screen but recurring payments still aren't showing in the Sales screen. I'll have to wait until the next sale to see if this updates the current totals. I don't think anything else on my site has really changed that could cause a problem. Looking at the json output from the EDD-API in a browser window seems to show the correct information.

**Comment by cklosowski** [comment]
@jackmcconnell I don't believe the renewal payments have shown up in the past, but the totals deff have included them.

If we want we can generate a new issue for renewals to show up, but I believe that is going to be a case of Recurring needing to filter the statuses, not EDD itself, or the iOS app.

@pippinsplugins can you confirm if renewal orders show in the sales list in the iOS app?

**Comment by pippinsplugins** [comment]
@cklosowski auto renewals have never been listed in the sales list as far as I can recall.

**Comment by cklosowski** [comment]
@pippinsplugins perfect thanks.

So we can work on getting an update in to EDD Recurring to possibly show those in the sales list @jackmcconnell however that is not something that can get fixed in the iOS app.

If you're earnings totals are matching what you are expecting I think we can close this one out as a stale local data cache.

**Comment by jackmcconnell** [comment]
@cklosowski Sure, that sounds fine. It would be great if all the sales through EDD did show in the iOS app if possible: initial sales, recurring subscriptions, products where you can choose your price, and so on. 

I did notice one other thing that was: if you tap on a sale in the Sales screen, the app crashes every time, even after re-installing or restarting the device.
